roles/ovirt-repositories/tasks/backup-repos.yml

```diff
@@ -0,0 +1,12 @@
+---
+- name: creating directory repo-backup for repo files
+  file:
+    path: "{{ repos_backup_path }}"
+    state: directory
+  tags: backup
+
+- name: copy content of yum.repos.d to backup folder
+  shell: "cp /etc/yum.repos.d/*.repo {{ repos_backup_path }}"
```

Comment:
why not copy module? I think it can even detect changes, so you want copy the files there over and over, rather, only when needed.

Reply:
I think he wants to use local copy but copy module copies from local executing machines to the ansible hosts. Correct me if I am wrong. Checking alternatives.